src/app/services/mail/mail.service.ts

```diff
@@ -1,4 +1,4 @@
-import { tracer } from 'dd-trace'
+import tracer from 'dd-trace'
```

Comment:
is this change intended?

Reply:
oops no, but it shouldn't make a difference! it was probably introduced when I reinstated the tracer spans. But it still works, tested on staging + the default import was already used in `mail.utils.ts` before